appservice/src/SiteWrapper.ts

```diff
@@ -584,14 +590,18 @@ export class SiteWrapper {
             isMercurial: false
         };
 
-        this.log(outputChannel, `"${this.appName}" is being connected to GitHub repo. This may take several minutes...`);
+        this.log(outputChannel, `"${this.appName}" is being connected to the GitHub repo. This may take several minutes...`);
         try {
-            await client.webApps.createOrUpdateSourceControlWithHttpOperationResponse(this.resourceGroup, this.name, siteSourceControl);
+            await this.updateSourceControl(client, siteSourceControl);
```

Comment:
Do you think we'll always have to do a resync after updating source control? In other words, should updateSourceControl handle the resync or is it fine to have connectToGitHub handle the resync?

Reply:
The resync seems to be specifically an issue for connecting to GitHub.  When we support other sourceControls (such as Bitbucket), I think we can see if that resync will be necessary for every updateSourceControl.